src/ceph-volume/tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py27, py35, py36, flake8
 deps=
   pytest
   mock
+basepython = python
```

Comment:
Hm I was reading up on this setting and I'm not sure this is what we want. iirc this uses whatever the `python` binarz points to instead of the env we specified.

We should probably just backport https://github.com/ceph/ceph/pull/28060/commits/f1db0a864814d9b81192d89b6eebb6c624ddec3b

Feel free to do this in this PR if you agree.